pkg/provisioners/inmemory/channel/reconcile.go

```diff
@@ -130,133 +119,30 @@ func (r *reconciler) reconcile(ctx context.Context, c *eventingv1alpha1.Channel)
 
 	// We are syncing three things:
```

Comment:
Renumber the list and put the correct number in this comment (or remove the number from this comment altogether).